### DISCUSSION TITLE
Does 0.2.0... work for anyone?

### DISCUSSION THREAD

**Discussion by no3rdw** [joedevivo/vscode-circuitpython#140] [General]
When I have v0.1.20 installed, I can use this extension to select a COM port and open the serial monitor.  When I upgrade to 0.2.0 and try to open the serial monitor, or select _any_ functionality listed when hitting Control-Shift-P, it just displays the message 'command 'circuitpython.(commandName)' not found (replace commandName with selectSerialPort, openSerialMonitor, etc).

When I revert to 0.1.20, everything goes back to working.  What am I doing wrong?

**Comment by moviebrain** [comment]
It does not work for me at all. Reinstall 1.20 and everything is perfect

**Reply by nickaxgit** [reply]
same

**Comment by wmerkens** [comment]
Yes, 0.2.0 will work. Please take a look at issues #151 and #148 for comments and instructions on fixing the build-stubs.sh to account for a change in the Adafruit circuitpython repo download and usage.

**Comment by wmerkens** [comment]
I have decided to fork this project to do the fixes myself and push a newer version to the VSC Marketplace. The repo is https://github.com/wmerkens/vscode-circuitpython, and on the VSC marketplace, look for wmerkens/circuitpython V2. My current stable version is V0.2.5, and I have a test version, V0.2.6-test, that incorporates a newer board selection list; I modified how it builds that list.
